client/src/partials/finance/reports/cash_payment/templates/grid.canceled.tmpl.html

```diff
@@ -0,0 +1,12 @@
+<!-- This template is taken from https://raw.githubusercontent.com/angular-ui/ui-grid/master/src/templates/ui-grid/ui-grid-row.html -->
+
+<div
+  ng-repeat="(colRenderIndex, col) in colContainer.renderedColumns track by col.uid"
+  ui-grid-one-bind-id-grid="rowRenderIndex + '-' + col.uid + '-cell'"
+  class="ui-grid-cell"
+  ng-class="{ 'ui-grid-row-header-cell': col.isRowHeader, 'strike' : row.entity.type_id === grid.appScope.bhConstants.transactionType.CREDIT_NOTE }"
+  data-vals="{{col.isRowHeader}}"
+  role="{{col.isRowHeader ? 'rowheader' : 'gridcell'}}"
+  ng-style="row.entity.type_id === grid.appScope.bhConstants.transactionType.CREDIT_NOTE ? {'background-color': '#ffb3b3'} : {'background-color':'none'}"
```

Comment:
I propose that you do this check in Javascript, for efficiency.  Like this:

```js
var reversedBackgroundColor = {'background-color': '#ffb3b3' };
var regularBackgroundColor = { 'background-color': 'none' };
rows.forEach(function (row) {
  row._backgroundColor =
    (row.type_id === bhConstants.transactionType.CREDIT_NOTE) ?  reversedBackgroundColor : regularBackgroundColor;
});
```
```html
ng-style="row.entity._backgroundColor"
```

The code is a bit less clear, but it would speed up the HTML/JS by a lot.  Our `ui-grid`s are currently really inefficient.